package.json

```diff
@@ -12,6 +12,9 @@
         "branch": "",
         "SHA": ""
     },
+    "dependencies": {
+        "npm": "2.15.10"
```

Comment:
Yeah, good catch on that one. I'd love to get rid of both of those ancient dependencies at the same time 👍